reddit/reddit-connector/reddit-connector-1.0.0/org.wso2.carbon.connector/src/test/resources/artifacts/ESB/config/restRequest/reddit/getAccessToken.json

```diff
@@ -0,0 +1,7 @@
+{
+  "refreshToken":"42591910-ITXmTVhxqv5NUlE_FPQ_YiXxlQM",
+  "grantType":"refresh_token",
+  "clientId":"bCzdckhMAJ91sw",
+  "clientSecret":"3uBELYRHhZLPUznD3D8Tj1OpqwE"
+}
+
```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/commit/c61da2ff6de0ced270ed9698ed4fbd8227a92f5b